api/src/controllers/login.js

```diff
@@ -235,4 +265,5 @@ module.exports = {
         return res.send();
       });
   },
+  _reset: () => loginTemplate = null // exposed for testing
```

Comment:
nit: on first read this is a confusing function, I think because I presume that in a single "line/action" closure the action would be an expression (ie return something) as opposed to being a statement.

If you feel neutral about it I think:

```suggestion
  _reset: () => { loginTemplate = null; } // exposed for testing
```

Would be easier to read, at least for me.

Reply:
Done.